src/Search.php

```diff
@@ -921,7 +915,7 @@ protected function set_results_context( &$context ) {
 
 				$context['page_types'][ $p4_post_type_id ] = [
 					'term_id' => $p4_post_type_id,
-					'name'    => $p4_post_type->name,
+					'name'    => $p4_post_type ?? $p4_post_type->name,
```

Comment:
Should be 
```php
$p4_post_type->name ?? null;
```
otherwise it would start using the whole object instead of the name.

Reply:
:facepalm:  and this is why you shouldn't cook & code